From 5e395b3f1ad99c5de2197b218d7a05eedb12c018 Mon Sep 17 00:00:00 2001
From: ranhui_xia <ranhui_xia@realsil.com.cn>
Date: Fri, 13 Dec 2013 12:30:12 +0800
Subject: [PATCH 3/3] Realtek BT: Add descrition for opp rx file types

If you want to make your platform support more files when receive
files by opp, you need to mofify by descriptions in patch.

Signed-off-by: ranhui_xia <ranhui_xia@realsil.com.cn>
---
 src/com/android/bluetooth/opp/Constants.java | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/src/com/android/bluetooth/opp/Constants.java b/src/com/android/bluetooth/opp/Constants.java
index 29df875..442cfa8 100644
--- a/src/com/android/bluetooth/opp/Constants.java
+++ b/src/com/android/bluetooth/opp/Constants.java
@@ -201,6 +201,20 @@ public class Constants {
         "virus/*",
     };
 
+    //REALTEK ADD START
+    /**
+    RTK: You could update here to support more mime types (file types)
+    e.g.
+    public static final String[] ACCEPTABLE_SHARE_INBOUND_TYPES = new String[] {
+        "*",
+    };
+    means support all mime types (file types)
+    But we suggest not to use this setting for secure issue
+    The better solution is to append the mime type, including aliases into this list
+    Mime types: http://help.dottoro.com/lapuadlp.php
+     */
+    //REALTEK ADD END
+
     /**
      * The MIME type(s) of we could accept from other device.
      * This is in essence a "white list" of acceptable types.
-- 
1.8.5.rc2

